project/src/Behaviors/GSTBlend.gd

```diff
@@ -1,27 +1,30 @@
 class_name GSTBlend
 extends GSTSteeringBehavior
 # Blends multiple steering behaviors into one, and returns acceleration combining all of them.
-
-# # Each behavior is associated with a weight - a modifier by which the result will be multiplied by,
+# 
+# Each behavior is associated with a weight - a modifier by which the result will be multiplied by,
```

Comment:
Another example of passive. Also, avoid separations such as ` - ` in formal or technical writing. A comma or another sentence would work here.